chanjo/cli/db.py

```diff
@@ -42,3 +43,46 @@ def remove(context, sample_id):
     LOG.info('delete sample (%s) from database', sample_id)
     store.session.delete(sample_obj)
     store.save()
+
+
+@db_cmd.command()
+@click.option('--group-id', '-g')
+@click.option('--sample-id', '-s')
+@click.option('--pretty', '-p', is_flag=True)
+@click.pass_context
+def samples(context, group_id, sample_id, pretty):
```

Comment:
What happens if samples=None or of the sample does not exists? Could you add a test or two?